src/Sylius/Behat/Client/ApiClientInterface.php

```diff
@@ -63,6 +63,10 @@ public function addRequestData(string $key, $value): void;
 
     public function addSubResourceData(string $key, array $data): void;
 
+    public function removeSubResource(string $subResource, int $key): void;
+
+    public function updateSubResource(string $subResource, int $key, array $value): void;
```

Comment:
I wonder, maybe we can extract a separate client just for subresource-related methods? *Or* just for index-related methods? I don't know if it should be done in this PR or in a separated one ⚖️

Reply:
After discussing on Slack we concluded that a sub-resource shouldn't be edited from the resource.